app/.env

```diff
@@ -5,4 +5,10 @@ APP_SECRET="a9806d99cd5c1503d8af30c963c9411f"
 
 ###> doctrine/doctrine-bundle ###
 DATABASE_URL="mysql://user:password@mariadb:3306/symfony-bootstrap"
-###< doctrine/doctrine-bundle ###
\ No newline at end of file
+###< doctrine/doctrine-bundle ###
+###> symfony/messenger ###
+# Choose one of the transports below
+# MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
+# MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
+MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
```

Comment:
Are you sure you want to use Doctrine transport? I think you don't, you want to use Symfony Messenger only with queueing in memory (type: sync).